x-pack/plugins/reporting/index.d.ts

```diff
@@ -13,6 +13,8 @@ import {
   NotificationsStart,
 } from '../../../src/core/public';
 
+export { ToastsSetup, HttpSetup, ApplicationStart } from 'src/core/public';
```

Comment:
np has restrictions on where these can be imported (I'm solely using types in places), only workaround I could find is passing them through our types file